official/transformer/transformer_main.py

```diff
@@ -457,7 +463,15 @@ def run_transformer(flags_obj):
       bleu_source=flags_obj.bleu_source,
       bleu_ref=flags_obj.bleu_ref,
       bleu_threshold=flags_obj.stop_threshold,
-      vocab_file_path=os.path.join(flags_obj.data_dir, flags_obj.vocab_file))
+      vocab_file=flags_obj.vocab_file)
+
+  if flags_obj.export_dir:
+    serving_input_fn = tf.estimator.export.build_raw_serving_input_receiver_fn({
+        "inputs": tf.placeholder(tf.int64, [None, None])
```

Comment:
You can use a TensorServingInputReceiver, and then you don't need to unwrap the dict above.